ivadomed/training.py

```diff
@@ -293,7 +293,7 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
                 torch.save(state, resume_path)
```

Comment:
Could you double check if `torch.save` supports pathlib objects references? 
Its signature seems to suggest more `str` or file object? 

![image](https://user-images.githubusercontent.com/6720068/130370768-fa007c4e-792c-4018-99b5-a87080cc85cb.png)